core/threads.go

```diff
@@ -496,23 +496,41 @@ func (t *Textile) addAccountThread() error {
 		}
 		// catch malformed account threads from 0.1.10
 		if x.Id == aid.Pretty() {
+
+			// catch schema-less account threads from 0.1.11
+			if x.Schema == nil {
```

Comment:
Sort of a migration that adds the avatar schema to the account thread if it's missing.